core/domain/feedback_jobs_one_off.py

```diff
@@ -99,3 +99,27 @@ def _cache_last_nonempty_message_author_id(
             thread_model.last_nonempty_message_author_id = message_author_id
             return True
         return False
+
+
+class GeneralFeedbackThreadUserOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for deleting GeneralFeedbackThreadUserModels with user_id
+    equal to None or 'None'.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
+        return [feedback_models.GeneralFeedbackThreadUserModel]
+
+    @staticmethod
+    def map(model_instance):
+        """Implements the map function for this job."""
+        if model_instance.user_id == 'None' or model_instance.user_id is None:
+            model_instance.delete()
+            yield ('SUCCESS - DELETED', model_instance.id)
+        else:
+            yield ('SUCCESS - NOT DELETED', model_instance.id)
+
+    @staticmethod
+    def reduce(key, values):
+        yield (key, len(values))
```

Comment:
Do we not want to keep the counts of the different keys as separate? Else I don't see the point of having two different labels.

Reply:
They are separate, they will get split automatically by the MapReduce system. You can see that in the tests.

Reply:
Oh sorry! I got confused... You are right...